src/omemo/omemo.c

```diff
@@ -791,6 +791,19 @@ omemo_on_message_send(ProfWin* win, const char* const message, gboolean request_
                 .device_id = GPOINTER_TO_INT(device_ids_iter->data)
             };
 
+            // Don't encrypt for this device (according to
+            // <https://xmpp.org/extensions/xep-0384.html#encrypt>).
+            // Yourself as recipients in case of MUC
+            Jid* me = jid_create(connection_get_fulljid());
```

Comment:
You could use `connection_get_barejid()` which returns a `char*` with the barejid. Then you don't need to build it yourself.

Reply:
Done